BoardTest.py

```diff
@@ -0,0 +1,50 @@
+import board
+import unittest
+
+
+class BoardTest(unittest.TestCase):
+    def test_delta(self):
```

Comment:
Worth adding a comment that delta does no error checking so that it is not necessarily the case that the result is a legal board value.